### DISCUSSION TITLE
RTSPS Support?

### DISCUSSION THREAD

**Discussion by nilava** [koush/scrypted#1473] [Ideas]
I have encryption turned on in my NVR, thus all streams are only available throups rtsps:// protocol.

While i have been using Frigate with rtsps. i found it nearly imposibble to use scrypted without modyfying the library to support rtsps.

Is there any plans to supports RTSPS in the future?

**Comment by koush** [comment]
rtsps works. unifi protect plugin uses it.

**Comment by nilava** [comment]
doest work with amcrest (Dahua) plugin. So motion detection needs to be done locally rather than utilizing the NVR events.
